collection/src/collection.rs

```diff
@@ -447,9 +452,9 @@ impl<T: Id<T>> CollectionWithId<T> {
     /// # Examples
     ///
     /// ```
-    /// # use transit_model::collection::*;
+    /// # use collection::*;
     /// # use std::collections::HashMap;
-    /// # fn run() -> transit_model::Result<()> {
+    /// # fn run() -> Result<()> {
```

Comment:
This one probably need `collection::Result` instead of just `Result`.

Reply:
Mmmm, not true what I just said. The above line being `use collection::*;` so `Resultè is already in scope.